### ISSUE TITLE
Search by Arabic, English or Bangla

### ISSUE THREAD

**Issue by oazabir** [oazabir/QuranApp#6]

**Comment by ATouhou** [comment]
The best search engine I have seen so far:
http://www.alfanous.org/en/

**Comment by oazabir** [comment]
Many thanks for the pointer. I will probably have to do something within the app as there's plan to make it as a local mobile app.

**Comment by ATouhou** [comment]
Agreed on should'nt be depend on API of other projects. Alfqnous is
open-source,  just search it in Github ^^
Maybe you can use something from it
Den 12/08/2015 12.23 AM skrev "Omar AL Zabir" notifications@github.com:
